translations/strings.yaml

```diff
@@ -343,3 +343,8 @@ serverUnavailableModal:
   headerText: Server unavailable
   bodyText: This server location is temporarily unavailable. Choose a new location.
   buttonLabel: Choose a new location
+
+mobileOnboarding:
+  signUp: Sign up
```

Comment:
Could we add something to the ID to explain that it's a button? e.g. `signupBtn`

Reply:
done